apps/shipments/tasks.py

```diff
@@ -28,3 +28,21 @@ def tracking_data_update(self, shipment_id, payload):
                             rate_limit=shipment.background_data_hash_interval,
                             action_type=AsyncActionType.TRACKING,
                             use_updated_by=False)
+
+
+@shared_task(bind=True, autoretry_for=(RPCError,),
+             retry_backoff=3, retry_backoff_max=60, max_retries=10)
+def telemetry_data_update(self, shipment_id, payload):
+    log_metric('transmission.info', tags={'method': 'shipments_tasks.tracking_data_update',
+                                          'module': __name__})
+    shipment = Shipment.objects.get(id=shipment_id)
+
+    rpc_client = RPCClientFactory.get_client()
+    signature = rpc_client.add_telemetry_data(shipment.storage_credentials_id,
+                                              shipment.shipper_wallet_id,
+                                              shipment.vault_id,
+                                              payload)
+    shipment.set_vault_hash(signature['hash'],
+                            rate_limit=shipment.background_data_hash_interval,
+                            action_type=AsyncActionType.TRACKING,
```

Comment:
Make another AsyncActionType and use that. Note that this may have an impact on our admin views and jobs/transactions/shipmenthistory endpoints so test those.